.github/workflows/CI.yml

```diff
@@ -12,7 +12,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        rust: [stable, 1.40.0]
+        rust: [stable, 1.42.0]
```

Comment:
Is there a reason for the MSRV bump?

We try to avoid bumping the minimum supported Rust version unless it's strictly necessary. If we do need to bump the MSRV, we'll also need to update the [listed minimum supported version](https://github.com/tokio-rs/tracing/tree/master/tracing#supported-rust-versions) throughout the documentation (it's in the lib.rs RustDoc and the readme for each crate).

Reply:
(sorry that's a big pile of work, but IMO, it's really important to ensure that the MSRV is documented and visible for users)

Reply:
Don't worry, it didn't take very long :) tracing uses the same 'minimum supported version is X' wording everywhere so I automated it:

```sh
rg 'version is 1\.40' -l | xargs sed -i 's/version is 1\.40/version is 1.42/'
```

Reply:
There's also a note at the top of the `lib.rs` docs and READMEs [at the end of the "Overview" section](https://github.com/tokio-rs/tracing/blob/master/tracing/README.md#overview) ("Compiler support: requires rustc ..."). Mind `rg`ing those as well? Thanks!

Reply:
Ok, I think I got them all this time :)
```
fd '.(md|rs)' | xargs rg '1\.40' -l | xargs sed -i 's/1\.40/1.42/'
```